### PR TITLE
Fixed bleeding players still dripping blood after having been turned to stone

### LINKED ISSUE #30492: Turn to stone while bleeding
(WEB REPORT BY: herrytheafroheadcrab REMOTE: 206.221.180.138:7777)
# Revision
2f6bcded66dad986d846ab7ed469038f01ec01a8

# Description
Getting turned to stone while having a bleeding external wound and still bleeding
# Steps to Reproduce
Get turned to stone by wiznerd or cockatrice
# What you Expected
To not bleed out while I am a statue and have time to bandage after I unstone
# What Actually Happened
Kept bleeding out while turned to stone

### PR DESCRIPTION
**Pull request by DeityLink**
Fixes #30492

:cl:
* bugfix: Mobs are now timestopped when turned to stone, which suspends things like bleeding and other ongoing effects. (herrytheafroheadcrab)
* tweak: Just like Wizard Flesh to Stone, Cockatrice petritricin now also turns compatible mobs into marble mannequins instead of statues, which means that items can be removed from them without breaking them.

### CODE REVIEW

File: code/modules/organs/blood.dm [hunk 1]
```diff
@@ -184,6 +184,8 @@ var/const/BLOOD_VOLUME_SURVIVE = 122
 		*/
 
 		//Bleeding out
+		if (istype(loc, /obj/structure/mannequin/new_mannequin)) // Petrified
```
Comment (DeityLink) [inline code/modules/organs/blood.dm]:
```suggestion
		if (istype(loc, /obj/structure/mannequin)) // Petrified
```

Comment (DeityLink) [reply code/modules/organs/blood.dm]: Test

**Comment by Kurfursten** [comment]
Wouldn't it make more sense to just timestop anyone who has been petrified until free?

**Comment by DeityLink** [comment]
Yeah agreed, I've streamlined petrification code so that both Cockatrice and Wizards end up calling turn_into_statue() which takes care of setting up the timestopped var.

**Pull request merged**